distribution/tools/java-version-checker/build.gradle

```diff
@@ -1,12 +1,27 @@
-apply plugin: 'elasticsearch.build'
+apply plugin: 'elasticsearch.java'
 
-targetCompatibility = JavaVersion.VERSION_1_7
+sourceSets {
+  unsupportedJdkVersionEntrypoint
+}
+
+tasks.named('compileUnsupportedJdkVersionEntrypointJava').configure {
+  targetCompatibility = JavaVersion.VERSION_1_7
+}
+
+tasks.named("jar") {
+  manifest {
+    attributes("Multi-Release": "true")
+  }
 
-// java_version_checker do not depend on core so only JDK signatures should be checked
-tasks.named('forbiddenApisMain').configure {
-  replaceSignatureFiles 'jdk-signatures'
+  from(sourceSets.unsupportedJdkVersionEntrypoint.output)
+  eachFile { details ->
+    if (details.path.equals("org/elasticsearch/tools/java_version_checker/JavaVersionChecker.class") &&
```

Comment:
I don't know why but I'm triggered by underscores in a package name :smile:

Reply:
I agree, I'm happy to change this, it was just already existing.

Reply:
We can definitely leave it as-is.